src/libsync/owncloudtheme.cpp

```diff
@@ -35,63 +33,13 @@ ownCloudTheme::ownCloudTheme()
 {
 }
 
-QString ownCloudTheme::configFileName() const
-{
-    return QLatin1String("owncloud.cfg");
-}
-
-QString ownCloudTheme::about() const
-{
-    QString devString;
-    devString = tr("<p>Version %2. For more information visit <a href=\"%3\">https://%4</a></p>"
-                       "<p>For known issues and help, please visit: <a href=\"https://central.owncloud.org/c/desktop-client\">https://central.owncloud.org</a></p>"
-                       "<p><small>By Klaas Freitag, Daniel Molkentin, Olivier Goffart, Markus Götz, "
-                       " Jan-Christoph Borchardt, and others.</small></p>"
-                       "<p>Copyright ownCloud GmbH</p>"
-                       "<p>Licensed under the GNU General Public License (GPL) Version 2.0<br/>"
-                       "ownCloud and the ownCloud Logo are registered trademarks of ownCloud GmbH "
-                       "in the United States, other countries, or both.</p>")
-                    .arg(Utility::escape(MIRALL_VERSION_STRING),
-                        Utility::escape("https://" MIRALL_STRINGIFY(APPLICATION_DOMAIN)),
-                        Utility::escape(MIRALL_STRINGIFY(APPLICATION_DOMAIN)));
-
-    devString += gitSHA1();
-    return devString;
-}
-
 #ifndef TOKEN_AUTH_ONLY
-QIcon ownCloudTheme::trayFolderIcon(const QString &) const
-{
-    QPixmap fallback = qApp->style()->standardPixmap(QStyle::SP_FileDialogNewFolder);
-    return QIcon::fromTheme("folder", fallback);
-}
-
-QIcon ownCloudTheme::applicationIcon() const
-{
-    return themeIcon(QLatin1String("owncloud-icon"));
-}
-
-
-QVariant ownCloudTheme::customMedia(Theme::CustomMediaType type)
+QVariant ownCloudTheme::customMedia(CustomMediaType)
 {
-    if (type == Theme::oCSetupTop) {
-        // return QCoreApplication::translate("ownCloudTheme",
-        //                                   "If you don't have an ownCloud server yet, "
-        //                                   "see <a href=\"https://owncloud.com\">owncloud.com</a> for more info.",
-        //                                   "Top text in setup wizard. Keep short!");
-        return QVariant();
-    } else {
-        return QVariant();
-    }
+    return QVariant();
 }
-
 #endif
```

Comment:
There's a `#ifndef`/`#endif` based on the same `TOKEN_AUTH_ONLY` wrapping this block already - will it make sense to remove this and [L59](https://github.com/owncloud/client/pull/6426/commits/b615b8ce57f32f052a3fcb43cf95334ac0dd785e#diff-5c233b67dd98ce7e8ea3ffbe0a4b3547R59) macros? Or is it semantically relevant to keep them?

Reply:
We should keep them, this eases when we sometimes build the cmd client without Qt Gui. (altough this is not officially supported, it works)